### ISSUE TITLE
Include with part of the JSON

### ISSUE THREAD

**Issue by nadavc-jfrog** [haoxins/gulp-file-include#187]
Consider the following JSON:
`[
  {
    "title": "My post title2",
    "description": "...",
    "imgSrc": "../../img/user-stories/rti-m.png"
  },
  {
    "title": "Another post",
    "description": "...",
    "imgSrc": "some-img2.png"
  },
  {
    "title": "One more post",
    "description": "....",
    "imgSrc": "some-img3.png"
  }
]`
Now, lets say I want to include just one of the records somewhere, how can I do that? for example:
@@include('some.html', "data.json[1]")

**Comment by ZhangChengLin** [comment]
Yes, is there a global variable that can be used when building a page? ?
